composer/workflows/dataproc_create_batch.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START composer_dataproc_create_batch]
+
+"""Example Airflow DAG that kicks off a batches job which will run a linear regression Spark ML job
+This DAG relies on an Airflow variable
+https://airflow.apache.org/docs/apache-airflow/stable/concepts/variables.html
+* project_id - Google Cloud Project ID to use for the Cloud Dataproc Serverless.
+* sparkml_file_location - Google Cloud Storage bucket where you've stored the natality_spark_ml file
+* phs_cluster - Google Dataproc Cluster, a single node cluster that you have started and is running
+TODO: Add the tutorial link once it is published.
+"""
+
+import datetime
+
+from airflow import models
+from airflow.providers.google.cloud.operators.dataproc import (
+    DataprocCreateBatchOperator, DataprocDeleteBatchOperator, DataprocGetBatchOperator, DataprocListBatchesOperator
+
+)
+from airflow.utils.dates import days_ago
+
+PROJECT_ID = "{{ var.value.project_id }}"
+REGION = "{{ var.value.region_name}}"
+BUCKET = "{{ var.value.bucket_name }}"
+PHS_CLUSTER = "{{ var.value.phs_cluster }}"
+METASTORE_CLUSTER = "{{var.value.metastore_cluster}}"
+DOCKER_IMAGE = "{{var.value.image_name}}"
+
+SPARK_ML_FILE_LOCATION = "gs://{{var.value.bucket_name }}/natality_sparkml.py"
+# for e.g.  "gs//my-bucket/natality_sparkml.py"
+# Start a single node Dataproc Cluster for viewing Persistent History of Spark jobs
+PHS_CLUSTER_PATH = \
+    "projects/{{ var.value.project_id }}/regions/{{ var.value.dataproc_region}}/clusters/{{ var.value.phs_cluster }}"
```

Comment:
```suggestion
    "projects/{{ var.value.project_id }}/regions/{{ var.value.region_name}}/clusters/{{ var.value.phs_cluster }}"
```
My bad - I messed this up and used two different names earlier